css-two-column-layout/styles.css

```diff
@@ -0,0 +1,54 @@
+.container {
+  max-width: 1000px;
+  min-width: 900px;
+  margin: auto;
+  font-family: "normal", Karla, sans-serif;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.column-full,
+.column-full h1,
+.column-full p {
+  width: 100%;
+  text-align: center;
+}
+
+.column-fourth {
+  flex-basis: 50%;
+  padding: 15px;
+}
+
+.column-images {
+  width: 30%;
+}
```

Comment:
You should name this `column-30`. `column-` classes should not be named after what they are applied to, they should be named after their size.